embedded-graphics/src/primitives/mod.rs

```diff
@@ -308,6 +308,51 @@ macro_rules! egline {
     }};
 }
 
+/// Create a [`Polyline`](./primitives/polyline/struct.Polyline.html) with optional styling using a
+/// convenient macro.
+///
+/// Note that styled polylines currently only support the `stroke_color` and `stroke_width` style
+/// properties. The `stroke_width` property is currently clamped at 1px. All other style properties
+/// are ignored.
```

Comment:
IMO this should be in the `Polyline` docs and not here, because it's easy to miss if this changes in the future.  And the macro doesn't use the mentioned properties directly, which could be confusing.

I would suggest to replace it by a note that that styling options are limited for polylines and that the `Polyline` docs provide more info.